package.json

```diff
@@ -13,6 +13,7 @@
         "start:server": "tinylicious"
     },
     "dependencies": {
+        "@fluid-experimental/data-objects": "^0.34.0-14942",
```

Comment:
Prob want to be consistent on version used, recommend not using 0.34 until release is finalized

Reply:
Yeah, 0.34 is coming out tomorrow. Just wanted to see if I could get this working. happy to wait on merge until 0.34 is out

Reply:
At that time probably also update the other dependencies to 0.34 too then.